graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -232,6 +265,9 @@ def _rename_name_and_add_to_record(self, node):
             return node
 
         new_name_string = self.renamings.get(name_string, name_string)  # Default use original
+        if new_name_string is None:
+            # suppress the type
```

Comment:
Style-matching: comments are generally capitalized in this file (and likely elsewhere in this repo)